src/wp-includes/class-wp-block-metadata-registry.php

```diff
@@ -166,6 +166,47 @@ public static function get_metadata( $file_or_folder ) {
 		return isset( $collection['metadata'][ $block_name ] ) ? $collection['metadata'][ $block_name ] : null;
 	}
 
+	/**
+	 * Gets the list of absolute paths to all block metadata files that are part of the given collection.
+	 *
+	 * For instance, if a block metadata collection is registered with path `WP_PLUGIN_DIR . '/my-plugin/blocks/'`,
+	 * and the manifest file includes metadata for two blocks `'block-a'` and `'block-b'`, the result of this method
+	 * will be an array containing:
+	 * * `WP_PLUGIN_DIR . '/my-plugin/blocks/block-a/block.json'`
+	 * * `WP_PLUGIN_DIR . '/my-plugin/blocks/block-b/block.json'`
+	 *
+	 * @since 6.8.0
+	 *
+	 * @param string $path The absolute base path for a previously registered collection.
+	 * @return string[] List of block metadata file paths, or an empty array if the given `$path` is invalid.
+	 */
+	public static function get_collection_block_metadata_files( $path ) {
+		$path = wp_normalize_path( rtrim( $path, '/' ) );
+
+		if ( ! isset( self::$collections[ $path ] ) ) {
+			_doing_it_wrong(
+				__METHOD__,
+				__( 'No registered block metadata collection was found for the provided path.' ),
+				'6.8.0'
+			);
+			return array();
+		}
+
+		$collection = &self::$collections[ $path ];
+
+		if ( null === $collection['metadata'] ) {
+			// Load the manifest file if not already loaded
```

Comment:
```suggestion
			// Load the manifest file if not already loaded.
```

Why PHPCS didn't show issue 🤔